docs/index.rst

```diff
@@ -61,6 +61,75 @@ Quick Start
 
    .. note:: It is always recommended to use python virtual environment
 
+.. _how_it_works:
```

Comment:
It works great locally, it seems to be an issue with how readthedocs processes these specific attributes.